cmd/controller-manager/app/controllermanager.go

```diff
@@ -596,6 +607,7 @@ func startApplicationFailoverController(ctx controllerscontext.Context) (enabled
 		Client:              ctx.Mgr.GetClient(),
 		EventRecorder:       ctx.Mgr.GetEventRecorderFor(applicationfailover.RBApplicationFailoverControllerName),
 		ResourceInterpreter: ctx.ResourceInterpreter,
+		RateLimiterOption:   ctx.Opts.RateLimiterOptions,
```

Comment:
It seems that you ignore setting `RateLimiterOption` for `crbApplicationFailoverController`.